src/restconfig/src/main/java/org/geoserver/rest/resources/resourceMetadata.ftl

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```

Comment:
The restlet TemplateRepresentation class sets the output character encoding the same as the input character encoding. So setting <#ftl encoding="utf-8"> should ensure both input as well as output encoding are utf-8 and I can do <meta charset="UTF-8"> as well

(The existing templates are not caring about this...)